admin/pages/xml-sitemaps.php

```diff
@@ -28,7 +28,13 @@
 
 $tabs = new WPSEO_Option_Tabs( 'sitemaps' );
 $tabs->add_tab( new WPSEO_Option_Tab( 'general', __( 'General', 'wordpress-seo' ), array( 'video_url' => 'https://yoa.st/screencast-sitemaps' ) ) );
-$tabs->add_tab( new WPSEO_Option_Tab( 'user-sitemap', __( 'User sitemap', 'wordpress-seo' ), array( 'video_url' => 'https://yoa.st/screencast-sitemaps-user-sitemap' ) ) );
+
+$all_options = WPSEO_Options::get_all();
+
```

Comment:
It it possible to use WPSEO_Options::get_option() instead of get_all(). It seems to only be used for disable-post-format?